gateway/coprocess.go

```diff
@@ -193,8 +193,12 @@ func (c *CoProcessor) ObjectPostProcess(object *coprocess.Object, r *http.Reques
 		values.Set(p, v)
 	}
 
-	r.URL, _ = url.ParseRequestURI(object.Request.Url)
+	r.URL, err = url.ParseRequestURI(object.Request.Url)
+	if err != nil {
+		return err
```

Comment:
having named return parameter and use it in `return` statement looks a little odd

Reply:
Updated